targets/audio

```diff
@@ -116,7 +116,8 @@ build_cras() {
         tar -xmf adhd.tar.gz cras/src
 
         cd cras/src
-
+	# Fixes #4506
+	sed -i 's/-Werror//g' Makefile.am
```

Comment:
we've been avoiding disabling -Werror since we do want to find out if something new in cras breaks on old GCCs.

Does it work to add `-Wno-attributes` to line 179 instead (and append to the comment preceding it)?